### PR TITLE
Feature/fill tbd fields

### PR DESCRIPTION
**Pull request by christian-rli**
Fills all TBD-fields in v1.3 and addresses #4 . Hope it doesn't conflict with #11 .

### CODE REVIEW

File: metadata/v130/schema.json [hunk 1]
```diff
@@ -164,22 +164,22 @@
                         "type": "string"
                     },
                     "fields": {
-                        "description": "Each field (column) of the table is described in detail. There are three sub keys: name, description, and unit for each column.",
+                        "description": "Each field (column) of the table is described in detail. Every field is described in an object",
                         "type": "array",
                         "items": {
-                            "description": "TBD",
+                            "description": "Each field object contains all three keys: name, description.",
```
Comment (Bachibouzouk) [inline metadata/v130/schema.json]: What is the third key?

File: metadata/v130/schema.json [hunk 2]
```diff
@@ -193,48 +193,48 @@
             "type": "string"
         },
         "_comment": {
-            "description": "TBD",
+            "description": "Object. The “_comment”-section is used as a self-description of the final metadata-file. It is text, intended for humans and can include a link to the metadata documentation(s), required value formats and similar remarks. The comment section has no fix structure or mandatory values, but a useful self-description, similar to the one depicted here, is encouraged.",
             "type": "object",
             "properties": {
                 "_url": {
-                    "description": "TBD",
+                    "description": "URL to the description if this metadata standard. Example: https://github.com/OpenEnergyPlatform/metadata/blob/develop/metadata/v130/schema.json",
```
Comment (Bachibouzouk) [inline metadata/v130/schema.json]: There is no verb in this sentence

File: metadata/v130/schema.json [hunk 3]
```diff
@@ -193,48 +193,48 @@
...
                     "type": "string"
                 },
                 "_additional_information": {
-                    "description": "TBD",
+                    "description": "Object containing additional tips on the metadata format.",
                     "type": "object",
                     "properties": {
                         "_dates": {
-                            "description": "TBD",
+                            "description": "Description of how dates should be used. Example: Dates must follow the ISO8601 (JJJJ-MM-TT)",
```
Comment (Bachibouzouk) [inline metadata/v130/schema.json]:
```suggestion
                            "description": "Description of how dates should be formatted. Example: Dates must follow the ISO8601 (JJJJ-MM-TT)",
```
?

File: metadata/v130/schema.json [hunk 4]
```diff
@@ -193,48 +193,48 @@
...
                             "type": "string"
                         },
                         "_units": {
-                            "description": "TBD",
+                            "description": "Description of how units should be used. Example: Use a space between Numbers and units (100 m)",
```
Comment (Bachibouzouk) [inline metadata/v130/schema.json]:
```suggestion
                            "description": "Description of how units should be formatted. Example: Use a space between Numbers and units (100 m)",
```

File: metadata/v130/schema.json [hunk 5]
```diff
@@ -193,48 +193,48 @@
...
                             "type": "string"
                         },
                         "_none": {
-                            "description": "TBD",
+                            "description": "Description of how empty calues should be used. Example: If not applicable use 'none'",
```
Comment (Bachibouzouk) [inline metadata/v130/schema.json]:
```suggestion
                            "description": "Description of how empty values should be formatted. Example: If not applicable use 'none'",
```

File: metadata/v130/schema.json [hunk 6]
```diff
@@ -193,48 +193,48 @@
             "type": "string"
         },
         "_comment": {
-            "description": "TBD",
+            "description": "Object. The “_comment”-section is used as a self-description of the final metadata-file. It is text, intended for humans and can include a link to the metadata documentation(s), required value formats and similar remarks. The comment section has no fix structure or mandatory values, but a useful self-description, similar to the one depicted here, is encouraged.",
```
Comment (Bachibouzouk) [inline metadata/v130/schema.json]:
```suggestion
            "description": "Object. The “_comment”-section is used as a self-description of the final metadata-file. It is text, intended for humans and can include a link to the metadata documentation(s), required value formats and similar remarks. The comment section has no fixed structure or mandatory values, but a useful self-description, similar to the one depicted here, is encouraged.",
```

File: metadata/v130/schema.json (hunk 1 above)
Comment (christian-rli) [inline metadata/v130/schema.json]:
```suggestion
                            "description": "Each field object contains all three keys: name, description and unit.",
```

Comment (christian-rli) [reply metadata/v130/schema.json]:
```suggestion
                            "description": "Each field object contains all three keys: name, description and unit.",
```

File: metadata/v130/schema.json (hunk 2 above)
Comment (christian-rli) [reply metadata/v130/schema.json]:
```suggestion
                    "description": "URL that refers to the description if this metadata standard. Example: https://github.com/OpenEnergyPlatform/metadata/blob/develop/metadata/v130/schema.json",
```

File: metadata/v130/schema.json [hunk 7]
```diff
@@ -66,7 +66,7 @@
             "description": "Contains information about the sources",
             "type": "array",
             "items": {
-                "description": "TBD",
+                "description": "Source object in list of source objects. Each object has all name-value-pairs.",
                 "type": "object",
                 "properties": {
```
Comment (christian-rli) [inline metadata/v130/schema.json]:
```suggestion
                "properties": {blub
```

**Pull request merged**